src/compiler/scala/tools/nsc/backend/jvm/BCodeBodyBuilder.scala

```diff
@@ -1354,4 +1368,14 @@ abstract class BCodeBodyBuilder extends BCodeSkelBuilder {
           ArrayBType(ObjectRef)),
         coreBTypes.jliCallSiteRef
       ).descriptor)
+
+  lazy val symbolLiteralBoostrapHandle =
+    new asm.Handle(asm.Opcodes.H_INVOKESTATIC,
+      "scala/runtime/SymbolLiteral", "bootstrap",
+      "(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/invoke/MethodType;Ljava/lang/String;)Ljava/lang/invoke/CallSite;")
```

Comment:
References to class names should go through a BType, see https://github.com/scala/scala/blob/fb22e2b0a73605d654c153e02d454e5cec21f355/src/compiler/scala/tools/nsc/backend/jvm/analysis/BackendUtils.scala#L74-L78